content/knowledge-base/sentry-crashlytics-dsym-uploading.md

```diff
@@ -0,0 +1,67 @@
+---
+description: How to upload dsym artifacts to Sentry
+title: Uploading dSYM to Sentry
+weight: 12
+---
+
+**Sentry** is a crash reporting platform that provides you with real-time insight into production deployments with info to reproduce and fix crashes.
+
+**dSYM** is used to symbolicate your crash reports. The purpose of **dSYM** is to replace symbols in the crash logs with the specific methods so it will be readable and helpful for debugging the crash. To generate debug symbols with **Sentry**, **sentry/react-native** dependency needs to be installed. **Sentry** provide different ways of installation processes depending on what platform your app is built with. More information can be found in the **Sentry** environment [here](https://sentry.io/welcome/)
```

Comment:
```suggestion
**dSYM** is used to symbolicate your crash reports. The purpose of **dSYM** is to replace symbols in the crash logs with the specific methods so it will be readable and helpful for debugging the crash. To generate debug symbols with **Sentry**,  a platform-specific dependency needs to be installed. **Sentry** provides different ways of installing the Sentry dependency according to which platform your app is built with. More information can be found in the **Sentry** documentation [here](https://docs.sentry.io/). 
```